lib/shared/addon/components/cluster-driver/driver-rke/template.hbs

```diff
@@ -332,7 +378,7 @@
               </ul>
             </li>
             <li>
-              {{t 'clusterNew.rke.command.instructions'}}
+              {{t (if isLinux 'clusterNew.rke.command.instructions' 'clusterNew.rke.command.winInstructions') htmlSafe=true}}
```

Comment:
I dont quite understand this? The linux option gives us a real command to run, while the windows option shows nothing which i mentioned in the component above.

Reply:
![image](https://user-images.githubusercontent.com/18737885/44615855-45798800-a876-11e8-8dfc-bf6f5cc2e154.png)

When you select windows, it will show this instruction to indicate which command should be running in CLI and windows version required.

Reply:
the command is now showing up correctly.